spring-boot/src/test/java/org/springframework/boot/logging/LoggingApplicationListenerTests.java

```diff
@@ -279,6 +280,23 @@ public void parseLevelsCaseInsensitive() throws Exception {
 	}
 
 	@Test
+	public void parseLevelsCaseInsensitivePassesTurkeyTest() throws Exception {
+		Locale original = Locale.getDefault();
```

Comment:
Nice idea but I think since we don't have that many tests like this we're probably better off with the existing solution rather than adding a dependency.